index.html

```diff
@@ -3,19 +3,60 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
+    <title> <img src="favicon.ico" alt="Karma logo">Karma</title>
     <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
     <!-- Add a link to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
+</head> 
+
+<header>
+</header>
+
+
+<nav>
```

Comment:
The nav needs to be inside the body tag.
The `head` tag wraps all meta information (not displayed on the page, just tells the browser about the page). Everything you want to _appear_ on the page should live inside the `body` tag